ProjectAnswers.tex

```diff
@@ -128,7 +128,51 @@ \subsection{Continuing Problem  1.2}
 
 
 \subsection{Prove Theorem 1.1}
-// incomplete
+\textbf{Theorem:} Let $A$ be a square matrix. Suppose all the eigenvalues of $A$ are real numbers and have distinct absolute values. Then the matrices $A_1, A_2, \dots, A_k, \dots$ approach an upper triangular matrix $U$ whose diagonal entries are the eigenvalues of $A$.
+
+\begin{proof}
+We proceed in the following steps:
+
+\textbf{Step 1: Schur Decomposition}  
```

Comment:
I think we should stick with theorem's we've learned in class

Reply:
Also if you used a proof from the internet to get you started, that's ok but you have to cite it.